blocks/editable/index.js

```diff
@@ -84,6 +85,13 @@ export default class Editable extends wp.element.Component {
 		};
 	}
 
+	onConfig( settings ) {
+		return ( this.props.onConfig || identity )( {
+			...settings,
+			forced_root_block: this.props.inline ? false : 'p',
```

Comment:
Should our `forced_root_block` always take precedence over the block's own? I don't really feel strongly either way, just seems like a block overriding settings might want ability to override _anything_.